bijection-scrooge/src/main/scala/com/twitter/bijection/scrooge/ScroogeCodecs.scala

```diff
@@ -35,10 +35,19 @@ object BinaryScalaCodec {
 }
 
 class BinaryScalaCodec[T <: ThriftStruct](c: ThriftStructCodec[T])
```

Comment:
Somehow I missed the new protocol implementation (was that left out of the first commit?) I get it now.